src/api/index.js

```diff
@@ -13,6 +14,14 @@ export const axios = create({
   baseURL,
   withCredentials: true,
   paramsSerializer: params => qs.stringify(params, { encode: false }),
+  transformRequest: [data => evolve({ birthYear: BirthYear.fromDisplay }, data)]
```

Comment:
You can write this point-free as `[evolve({ birthYear: BirthYear.fromDisplay })]`